src/AppBundle/Resources/translations/odr-assets.en.yml

```diff
@@ -106,6 +106,7 @@ form:
       label: What is %client%’s share of the property?
     isSubjectToEquityRelease:
       label: Is this property subject to an equity release scheme?
+      hint: This means that %client%'s stake, or equity, in the property has been used to generate a lump sum or income
```

Comment:
anytime there is a %client% variable that wasn't there, it might be not rendered